backend/functions/lib/use-cases/case-management.test.ts

```diff
@@ -219,13 +221,7 @@ describe('Case management tests', () => {
         },
       ];
 
-      // TODO: This is gross. The response wrapper should be added by the controller, not the use case.
-      // I Agree.  This is gross.
```

Comment:
Still gross. Still should be migrated to the controller. Can punt to another day since I'm certain this isn't the last time we will be working in this use case.